src/marketplace/eqp/v1/packages.md

```diff
@@ -90,11 +90,14 @@ Both `POST` and `PUT` requests support a batch model where multiple packages can
 |original_launch_date|DateTime|GET|-|yes|The UTC date and time this version of the package was first released to the store.|`YYYY-MM-DD HH:MM:SS`|
 |offset|integer|GET|-|no|In combination with the `limit` parameter, it can be used for paging the collection of packages.|See [Get package details](#get-package-details). Default value is 0.|
 |platform|string|GET, POST, PUT|technical|yes|The Magento platform compatibility of this package.|`M2`|
+|pricing_model|object|GET, POST|marketing|no|How to interpret the pricing for this package.|See [Object Details](#object-details)|
+|pricing_model.pricing_type|string|GET, POST|marketing|no|Which pricing model is used by this package.|`one-time`, `subscription`|
+|pricing_model.payment_period|string|GET, POST|marketing|no|For a package using the "one-time" payment model, the number `1` signifies "once."  For a subscription, how often (in terms of months) payments are due.  Currently, only annual subscriptions are supported.|`1` for "one-time" payments, or `12` for yearly subscriptions.|
 |prices|array|GET, POST, PUT|marketing|no|The list of prices in USD set for this package by edition, and the respective installation prices (if any). Editions must match `version_compatibility`.|Array of sub-objects.|
 |prices[N].currency_code|string|GET, POST, PUT|marketing|no|The currency code for this price|Currently only `USD`|
 |prices[N].edition|string|GET, POST, PUT|marketing|no|The Magento edition for this price|`CE`, `EE`, `ECE`|
-|prices[N].price|number|GET, POST, PUT|marketing|no|The value for the purchase price of this package|A number, with up to two decimal places, eg 123.45|
-|prices[N].installation_price|string|GET, POST, PUT|marketing|no|The value for the installation price of this package|A number, with up to two decimal places, eg 123.45|
+|prices[N].price|number|GET, POST, PUT|marketing|no|The value for the purchase price of this package.  For subscriptions, this is the periodic (ex: yearly) price.|A number, with up to two decimal places, eg 123.45|
```

Comment:
```suggestion
|prices[N].price|number|GET, POST, PUT|marketing|no|The value for the purchase price of this package. For subscriptions, this is the annual price.|A number, with up to two decimal places, eg 123.45|
```
Since only one subscription period is supported, might as well state it's the annual price.